src/custom-element.js

```diff
@@ -510,6 +558,26 @@ function createBaseCustomElementClass(win) {
       ));
     }
 
+    /**
+     * @return {!Promise}
+     */
+    build() {
```

Comment:
This is only called in V1 elements, right? We should probably mark it so that we' don't get people calling it.

Reply:
No. It's now completely transparent. It just does the right thing for V0 vs V1 internally - but it always does the right thing.